src/Mod/Fem/Gui/DlgSettingsFemGeneral.ui

```diff
@@ -312,6 +236,29 @@
           </item>
          </layout>
         </item>
+        <item>
+         <layout class="QVBoxLayout" name="verticalLayout_10">
+          <item>
+           <widget class="Gui::PrefCheckBox" name="cb_overwrite_solver_working_directory">
+            <property name="layoutDirection">
+             <enum>Qt::LeftToRight</enum>
+            </property>
+            <property name="text">
+             <string>overwrite solver working directory with the directory choosen above</string>
```

Comment:
@berndhahnebach should `overwrite` be capitalized?

Reply:
I will change this.